CONTRIBUTING.md

```diff
@@ -77,6 +77,44 @@ README for more details.
 - Once a pull request is good to go, the code reviewer is the one who merges
 it, then deletes the branch.
 
+## Front end architecture
+### CSS
+- The front end will use the Web Design Standard's components, imported through ruby, adding extra functionality that's missing.
+- Atomic CSS methodolgy will be used for layout, modifier classes.
+  - If a group of atomic classes is used more then once in a repeated pattern, separate it into it's own component
+  - Design atomtic classes to ensure that HTML classes don't have to be changed in more then one place with a design change.
+- Will update CSS libraries (such as the WDS) on a bi-monthly basis or more immediately if a feature is added that's needed.
+- Will use `js-`* prefixed classes for JavaScript hooks.
+- Will use `test-`* prefixed classes for testing hooks, mainly integration testing.
+- Uses the [18f linting config](https://raw.githubusercontent.com/18F/frontend/18f-pages-staging/.scss-lint.yml) for scss linting.
+- Use `scss-lint` which runs on the latest PR commits.
+- CodeClimate will notify on a PR on Github if there are any linting infractions.
+- The root font size should always be set to 100% by default and 1.4% when on mobile.
```

Comment:
I'm not sure I understand the bit about 1.4 on mobile -- i think base font sizes should get smaller on smaller devices; in any case, i think this bullet is a bit in the weeds for this higher level overview and would remove